netkan/netkan/indexer.py

```diff
@@ -119,6 +119,8 @@ def status_attrs(self, new=False):
             attrs['last_indexed'] = datetime.now(timezone.utc)
         if self.ErrorMessage:
             attrs['last_error'] = self.ErrorMessage
+        else:
```

Comment:
This is a little nit picky, but we can probably put this in our attrs block above with:
```python
'last_error': getattr(self, ErrorMessage, None)
```
Which is generally preferred to storing an empty string.

Reply:
Do we need `getattr`, or would `self.ErrorMessage` work? It does get set to `None` in `__init__`:

https://github.com/KSP-CKAN/NetKAN-Infra/blob/5d7db8839f2951c6086af889ec6ae5c9a593565c/netkan/netkan/indexer.py#L17

Reply:
Oh man. That's probably what I intended when I wrote this. Yeah, just self.ErrorMessage will do it. Good catch!

Reply:
The status allows for this field to be set to `None`

https://github.com/KSP-CKAN/NetKAN-Infra/blob/5d7db8839f2951c6086af889ec6ae5c9a593565c/netkan/netkan/status.py#L33

Reply:
OK, force pushed that change (I don't like to keep commits that are simply errors).